play/src/front/WebRtc/CoWebsite/BBBCoWebsite.ts

```diff
@@ -1,11 +1,8 @@
-import type CancelablePromise from "cancelable-promise";
-import { inExternalServiceStore } from "../../Stores/MyMediaStore";
-import { screenWakeLock } from "../../Utils/ScreenWakeLock";
-import { coWebsiteManager } from "../CoWebsiteManager";
+import { coWebsiteManager } from "../../Stores/CoWebsiteStore";
 import { SimpleCoWebsite } from "./SimpleCoWebsite";
 
 export class BBBCoWebsite extends SimpleCoWebsite {
-    private screenWakeRelease: (() => Promise<void>) | undefined;
+    public screenWakeRelease: (() => Promise<void>) | undefined;
```

Comment:
Remove this from the class and put a variable in the matching Svelte component to handle the ScreenWakeRelease locally.